resources/js/common/components/AchievementsListItem/AchievementsListItem.tsx

```diff
@@ -71,19 +70,29 @@ export const AchievementsListItem: FC<AchievementsListItemProps> = ({
           <div className="mb-0.5 flex justify-between gap-x-2">
             {/* Title */}
             <div className="-mt-2 mb-0.5 md:mt-0">
-              <a href={route('achievement.show', { achievement })} className="font-medium">
-                {title}
-                {game?.title ? ' ' : null}
-              </a>
-
-              {game?.title ? (
-                <Trans
-                  i18nKey="<1>from</1> <2>{{gameTitle}}</2>"
-                  components={{
-                    1: <span />,
-                    2: <AchievementGameTitle game={game} />,
-                  }}
-                />
+              <span className="mr-2">
+                <a href={route('achievement.show', { achievement })} className="font-medium">
+                  {title}
+                  {game?.title ? ' ' : null}
+                </a>
+
+                {game?.title ? (
+                  <Trans
+                    i18nKey="<1>from</1> <2>{{gameTitle}}</2>"
+                    components={{
+                      1: <span />,
+                      2: <AchievementGameTitle game={game} />,
+                    }}
+                  />
+                ) : null}
+              </span>
+
+              {/* Points */}
+              {achievement.points && achievement.pointsWeighted ? (
+                <span className="inline-flex gap-1 text-[12px]">
+                  <span>{`(${achievement.points})`}</span>
+                  <WeightedPointsContainer>{`(${achievement.pointsWeighted})`}</WeightedPointsContainer>
```

Comment:
Commas have been lost.

Old:
![image](https://github.com/user-attachments/assets/3d852abd-3cb7-4076-9044-4dd9d0f6e509)

New:
![image](https://github.com/user-attachments/assets/85751f5d-1045-4539-bebe-c65afa5796e5)

Reply:
Fixed in latest.